crates/db_schema/src/schema.rs

```diff
@@ -647,6 +649,14 @@ table! {
     }
 }
 
+table! {
+    language (id) {
+        id -> Int4,
+        code -> Text,
+        name -> Text,
+    }
+}
+
```

Comment:
After local_user_language is in its own table, I think there'll be more`joinable!` lines added somewhere in this file.

IE 

language.id -> post.language_id
language.id -> local_user_language.language_id
local_user_language.id -> local_user.local_user_id

Reply:
fixed (i think)